### ISSUE TITLE
Image display cleanup, mostly on mobile.

### ISSUE THREAD

**Issue by RonanHevenor** [thepoly/polymer#74]
The following changes are the only changes that should be made to both mobile and desktop. Alt text on all images should be the image title. Also we should show photo credits and captions the same way we do inside articles when images appear on the homepage.

The remaining changes should be made only to mobile. Currently in articles when we have a caption on an image it makes it not full screen like it should be. All images in articles should take up the whole screen and if there's a caption then show it the same way we currently do. Just with the image being full. Also, on mobile and this is the last thing, we should have lines between each article block. Same color lines that we have on desktop on the homepage, just between each article on mobile. Make sure you don't do duplicate lines when it's the start of a section on the homepage.
